src/windows.ts

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable */
+
+declare global {
+  interface Window {
+    _env_: any;
+  }
+}
+
+export const BASE_APP_URL = window._env_
+  ? window._env_.REACT_APP_URL
+  : process.env.REACT_APP_URL;
+
+export const REACT_APP_CLIENT_ID = window._env_
```

Comment:
No need to use RACT_APP_ prefix for constants' names. It is necessary only for env variables' names.